lib/v2.yaml

```diff
@@ -81,10 +81,9 @@ analyzer:
     duplicate_shown_name: warning
     unused_catch_clause: warning
     unused_catch_stack: warning
-    # Disabled during the Dart 2.13 transition
```

Comment:
Are these actually cleared to be enabled again?

Reply:
I think it's fine to enable again in V2 since it will be opt-in. We just didn't want to block migration to 2.13.4 since the `unsafe_html` lint was updated to be more aggressive.